packages/keystone/bin/loading.html

```diff
@@ -128,37 +128,44 @@ <h1>Keystone is <span id="status">loading...</span></h1>
 
       function checkAndSetStatus() {
         let cancelled = false;
-        fetch('/', { headers: { Accept: 'application/json' } })
-          .then(result => result.json())
-          .catch(error => {
-            // We can get back an error "Cannot parse JSON" when a HTML
-            // response is returned, so we assume the server is ready
-            onReady();
+        if (!ready) {
+          fetch(window.location.href.replace(window.location.origin, ''), {
```

Comment:
Attempt to load the current browser URL instead of `/` which might not be mapped to any route depending on the application's config.